docs/manuals/source/DeveloperGuide/mediaformat.rst

```diff
@@ -592,7 +580,7 @@ Overall Storage Format
        |-------------------------------------------------------|
        |              JobLevel         (uint32_t)              |
        |-------------------------------------------------------|
-       |              FileSetMD5       (50 bytes)   VerNum 11  |
+       |              FileSetMD5       (128 bytes)             |
```

Comment:
Is the 50 Bytes wrong in the old docs?

Reply:
Yes and no.
```
struct Session_Label {
...
  char FileSetMD5[MAX_NAME_LENGTH]{};
};
```
So the md5 can have up to 127+1 bytes, but our `SerialString` routine only writes out the actual string data.
As Md5 is guaranteed to be 32 + 1 bytes long, i guess the real answer is 33 bytes.  Im not sure where the 50bytes comes from though.